rust/repositories.bzl

```diff
@@ -396,6 +397,8 @@ Args:
   toolchain_name_prefix: The per-target prefix expected for the rust_toolchain declarations
   version: The version of the tool among "nightly", "beta', or an exact version.
   iso_date: The date of the tool (or None, if the version is a specific version).
+  rustfmt_version: The version of rustfmt to be associated with the toolchain.
```

Comment:
nit: Can the order of the args match the attrs list?

Reply:
Done.